<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QMainWindow" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>681</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(64,66,68,120);

border-radius:8px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>191</width>
      <height>342</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="styleSheet">
        <string notr="true">QComboBox {
    border: 1px solid white;   
    border-radius: 3px;   
    padding: 1px 18px 1px 3px;   
    color: #fff;
    font: normal normal 15px &quot;Microsoft YaHei&quot;;
    background: transparent;
}
/* 下拉后，整个下拉窗体每项的样式 */
QComboBox QAbstractItemView::item {
    height: 30px;   /* 项的高度（设置pComboBox-&gt;setView(new QListView());后，该项才起作用） */
	background-color: rgba(100,106,100,150);
	color: rgb(64,64,64);
}
QComboBox QAbstractItemView::item:hover{
	background-color: rgba(255,255,191,100);
	color: rgb(64,64,64);
}
/* QComboBox中的垂直滚动条 */
QComboBox QAbstractScrollArea QScrollBar:vertical {
    width: 10px;
    background-color: #d0d2d4;   /* 空白区域的背景色  灰色green */
	border-radius: 3px;   
}


</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="AccountEdit">
       <property name="styleSheet">
        <string notr="true">background-color : transparent;
border-color: : transparent;
color : rgb(255,255,255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PasswdEdit">
       <property name="styleSheet">
        <string notr="true">background-color : transparent;
border-color: : transparent;
color : rgb(255,255,255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="RealnameEdit">
       <property name="styleSheet">
        <string notr="true">background-color : transparent;
border-color: : transparent;
color : rgb(255,255,255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="IdentifyEdit">
       <property name="styleSheet">
        <string notr="true">background-color : transparent;
border-color: : transparent;
color : rgb(255,255,255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="TelephoneEdit">
       <property name="styleSheet">
        <string notr="true">background-color : transparent;
border-color: : transparent;
color : rgb(255,255,255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DiplomaEdit">
       <property name="styleSheet">
        <string notr="true">background-color : transparent;
border-color: : transparent;
color : rgb(255,255,255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ProfessionEdit">
       <property name="styleSheet">
        <string notr="true">background-color : transparent;
border-color: : transparent;
color : rgb(255,255,255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="WorkTimeEdit">
       <property name="styleSheet">
        <string notr="true">background-color : transparent;
border-color: : transparent;
color : rgb(255,255,255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>121</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;UserType：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Username">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;UserName：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Password">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;Password：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Realname">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;RealName:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Identify">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;Identify：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;Telephone:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;Diploma:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;Profession:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;WorkTime:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Register">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
                   background-color:rgba(220,220,220,120);
                   border-style:outset;                                                             
                   border-radius:5px;                     
                   font: 75 15pt &quot;华文行楷&quot;;           
                   padding:6px;
                   }

                   QPushButton:pressed{
                   background-color:rgba(100,100,100,180);
                   border-color:rgba(100,100,100,30);
                   border-style:inset;
                   }
                   QPushButton:hover{
                   background-color:rgba(100,255,100,100);
                   border-color:rgba(255,255,255,200);
}
</string>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
                   background-color:rgba(220,220,220,120);
                   border-style:outset;                                                             
                   border-radius:5px;                     
                   font: 75 15pt &quot;华文行楷&quot;;           
                   padding:6px;
                   }

                   QPushButton:pressed{
                   background-color:rgba(100,100,100,180);
                   border-color:rgba(100,100,100,30);
                   border-style:inset;
                   }
                   QPushButton:hover{
                   background-color:rgba(100,255,100,100);
                   border-color:rgba(255,255,255,200);
}
</string>
       </property>
       <property name="text">
        <string>Concel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="warning">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>480</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : transparent;
color : rgb(117,247,88);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>141</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="Picture">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
                   background-color:rgba(220,220,220,120);
                   border-style:outset;                                                             
                   border-radius:5px;                     
                   font: 75 15pt &quot;华文行楷&quot;;           
                   padding:6px;
                   }

                   QPushButton:pressed{
                   background-color:rgba(100,100,100,180);
                   border-color:rgba(100,100,100,30);
                   border-style:inset;
                   }
                   QPushButton:hover{
                   background-color:rgba(100,255,100,100);
                   border-color:rgba(255,255,255,200);
}
</string>
       </property>
       <property name="text">
        <string>Picture</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="DisplayWidget" name="picture" native="true"/>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Camera">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
                   background-color:rgba(220,220,220,120);
                   border-style:outset;                                                             
                   border-radius:5px;                     
                   font: 75 15pt &quot;华文行楷&quot;;           
                   padding:6px;
                   }

                   QPushButton:pressed{
                   background-color:rgba(100,100,100,180);
                   border-color:rgba(100,100,100,30);
                   border-style:inset;
                   }
                   QPushButton:hover{
                   background-color:rgba(100,255,100,100);
                   border-color:rgba(255,255,255,200);
}
</string>
       </property>
       <property name="text">
        <string>Camera</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="breifIntroduceEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;华文行楷&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="breifIntroduce">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>310</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;BreifIntroduce:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>904</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayWidget</class>
   <extends>QWidget</extends>
   <header location="global">displaywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
